src/core/core.helpers.js

```diff
@@ -723,7 +723,7 @@ module.exports = function(Chart) {
 	};
 	helpers.addEvent = function(node, eventType, method) {
 		if (node.addEventListener) {
-			node.addEventListener(eventType, method);
+			node.addEventListener(eventType, method, {passive: true});
```

Comment:
It's not supported in [IE and Edge](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Browser_compatibility) and therefore `options` support needs to be [detected](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Safely_detecting_option_support) in order to call `addEventListener` with the correct arguments.